apps/pwabuilder/src/script/services/auth_service.ts

```diff
@@ -0,0 +1,137 @@
+import { PublicClientApplication, SilentRequest, AuthenticationResult, Configuration, AccountInfo, RedirectRequest, EndSessionRequest } from "@azure/msal-browser";
```

Comment:
love how you packaged this into a class, totally makes sense for this use case

Reply:
@amrutha95 ^^^